entry_types/scrolled/lib/tasks/pageflow_scrolled_tasks.rake

```diff
@@ -90,7 +90,7 @@ namespace :pageflow_scrolled do
             }
           },
           options: {
-            skip_encoding: true
+            skip_encoding: ENV.fetch('PAGEFLOW_SKIP_ENCODING_STORYBOOK_FILES', true)
```

Comment:
The string "false" will be truthy.

Reply:
Maybe false should be the default? As a developer I will probably want a working storybook, right? Then CI can use the env var to override?

Reply:
True, especially since the PR also changes the content elements to use the transcoded files.